components/playlist/browser/resources/components/playlistFolder.tsx

```diff
@@ -58,9 +58,10 @@ const StyledEditActionContainer = styled.div`
   border-bottom: 1px solid ${color.divider.subtle};
 `
 
-const StyledEditButton = styled(LeoButton)`
+const StyledEditButton = styled(LeoButton)<{ isDisabled?: boolean }>`
```

Comment:
`isDisabled` doesn't seem to be used

Reply:
oops, thanks.

FYI, `isDisabled` was not accessible without this. 

<img width="761" alt="image" src="https://github.com/brave/brave-core/assets/5474642/abf4aec0-fe2a-4025-9b23-1053131bab44">

And I tried this one too, but didn't work
<img width="734" alt="image" src="https://github.com/brave/brave-core/assets/5474642/4bf05715-0c7a-4dc5-9624-2d831a4f81e1">

I'm not sure what's happening 😥